backend/src/graphql/resolvers/TransactionResolver.ts

```diff
@@ -216,13 +216,9 @@ export class TransactionResolver {
     @Args() { currentPage = 1, pageSize = 25, order = Order.DESC }: Paginated,
     @Ctx() context: any,
   ): Promise<TransactionList> {
-    // get public key for current logged in user
-    const result = await apiGet(CONFIG.LOGIN_API_URL + 'login?session_id=' + context.sessionId)
-    if (!result.success) throw new Error(result.data)
-
     // load user
     const userRepository = getCustomRepository(UserRepository)
-    const userEntity = await userRepository.findByPubkeyHex(result.data.user.public_hex)
+    const userEntity = await userRepository.findByPubkeyHex(context.pubKey)
```

Comment:
Are you up to date to have the pubKey in the context object?

Reply:
Yes, I have merged your changes with the jwt token.